pkg/plugin/delete_pvc_action_plugin.go

```diff
@@ -0,0 +1,18 @@
+package plugin
+
+import (
+	"github.com/sirupsen/logrus"
+	velerov1api "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+	"github.com/vmware-tanzu/velero/pkg/plugin/velero"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+// PVCDeleteItemAction is a delete item action plugin for Velero.
+type NewPVCDeleteItemAction struct {
+	Log logrus.FieldLogger
+}
+
+func (p *NewPVCDeleteItemAction) Execute(item runtime.Unstructured, backup *velerov1api.Backup) (runtime.Unstructured, []velero.ResourceIdentifier, error) {
```

Comment:
Here are draft APIs of DeleteItemAction

```
// DeleteItemAction is an actor that performs an action based on an item in a backup that is being deleted.
type DeleteItemAction interface {
	// AppliesTo returns information about which resources this action should be invoked for.
	// A DeleteItemAction's Execute function will only be invoked on items that match the returned
	// selector. A zero-valued ResourceSelector matches all resources.
    AppliesTo() (ResourceSelector, error)

	// Execute allows the ItemAction to perform arbitrary logic with the item being deleted.
    Execute(DeleteItemActionInput) error
}
```
There is no return value other than error in the `Execute` func